presto-docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -0,0 +1,153 @@
+=================
+Phoenix Connector
+=================
+
+The Phoenix connector allows querying data stored in Phoenix.
+
+Compatibility
+-------------
+
+Connector is compatible with all Phoenix versions starting from 4.13.x.
+
+Installation
+------------
+
+To install a pre-built phoenix, use these directions:
+
+* Download and expand the latest phoenix-[version]-bin.tar.
+* Add the phoenix-[version]-server.jar to the classpath of all HBase region server and master and remove any previous version. An easy way to do this is to copy it into the HBase lib directory.
+* Restart HBase.
+* Add Phoenix connector in Presto.
+
+Note that this uses apache hbase version. If your HBase cluster is using CDH version,
+you'll install CDH compatibility version and recompile Phoenix connector.
+You can change the version by changing the `` dep.phoenix.version`` variable in pom.xml.
```

Comment:
I do not think user is going to modify anything in any `pom.xml` file from Presto codebase